packages/admin/src/pages/super-admin-dashboard/Navigation.test.tsx

```diff
@@ -0,0 +1,41 @@
+import '@testing-library/jest-dom';
+import { render } from '@testing-library/react';
+import Navigation from './Navigation';
+import { Role } from './types';
+
+jest.mock('next/config', () => () => ({
+  publicRuntimeConfig: {
+    FIND_A_GRANT_URL: 'mocked-find-a-grant-url',
+    APPLICANT_DOMAIN: 'mocked-applicant-domain',
+    TECHNICAL_SUPPORT_DOMAIN: 'mocked-technical-support-domain',
+  },
+}));
```

Comment:
this is mocked globally - add TECHNICAL_SUPPORT_DOMAIN there instead. setupJestMock is the file you're after